pages.de/common/git-clone.md

```diff
@@ -19,6 +23,10 @@
 
 `git clone -q {{url_zu_repository}}`
 
-- Klone ein existierendes Repository und rufe nur die neusten 10 Commits im standardmäßigen Branch ab (spart Zeit):
+- Klone ein existierendes Repository und rufe nur die neusten 10 Commits im Standard-Branch ab (spart Zeit):
 
 `git clone --depth {{10}} {{url_zu_repository}}`
+
+- Klone ein existierendes Repository, aber hole nur einen bestimmten Branch:
```

Comment:
`hole` sounds a bit *too* informal, imho.
```suggestion
- Klone ein existierendes Repository, aber lade nur einen bestimmten Branch herunter:
```
This might be a possibility. 🤷🏻‍♀️